realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1009,10 +1162,36 @@ public double sumFloat(String fieldName) {
     // Average
 
     /**
+     * Returns the average of a given field.
+     *
+     * @param fieldName  The field to calculate average on. Only properties of type int,
+     *                   float and double are supported.
+     * @return           The average for the given field amongst objects in an RealmList. This
```

Comment:
Done. The original description is wrong, should be "This will be of type double for all types of number fields."